code/__DEFINES/xenoartifact_materials.dm

```diff
@@ -0,0 +1,19 @@
+//Xenoartifact defines
+//Material defines. Used for characteristic generation.
+#define BLUESPACE "#1e7cff" //Silly toys
+#define PLASMA "#ff00c8" //Associated weapons
+#define URANIUM "#00ff0d" //Broken Enigmas
+#define BANANIUM "#ffd900" //Wildcard, may god have mercy. Was originally called BANANIUM, changed for localisation.
+#define DEBUGIUM "#ff4800"
+
```

Comment:
this will get confusing in the future. Can you somewhat clarify that those are the xenoartifact defines?
I'd suggest "XENOA_COLOR_BLUESPACE" for example.